src/components/filter/filter.e2e.ts

```diff
@@ -145,7 +145,7 @@ describe("calcite-filter", () => {
             name: "Matt",
             description: "developer",
             value: "matt",
-            metadata: { haircolor: "black", favoriteBand: "unknown" }
+            metadata: { haircolor: "black", favoriteBand: "Radiohead" }
```

Comment:
⚡ 🤘🏼 :neckbeard: 🤘🏼 ⚡

Reply:
Had to be done.

Reply:
![giphy](https://user-images.githubusercontent.com/1231455/157128769-afb49335-dc35-4974-b940-082746b330e9.gif)